src/main/java/frc/robot/Drive.java

```diff
@@ -134,32 +135,32 @@ public static void initializeDrive(){
     }
    
    public static void arcadeDrive(double side, double forward, boolean isShifted){
-        R1.set(side - forward);
-		L1.set(side + forward);
-		Shiftersint(isShifted);
-	}
-	
-	public static void Shiftersint (boolean isShifted){
-		Shifters = new DoubleSolenoid(ShiftersForward, ShiftersReverse);
-		if (isShifted == true){
-			Shifters.set(Value.kReverse);
-		}else {
-			Shifters.set(Value.kForward);
-		}
-	}
-	public static void setShifters(){
-		if (driveStick.getRawButton(9)){
-			Shiftersint(true);
-		}else{
-			Shiftersint(false);
-		}
+        R1.set(side + forward);
+		L1.set(side - forward);
```

Comment:
Again looks like indention issues?

Did we verify that this needed to change? I'm not sure how this change is related to the climber? Want to avoid breaking existing functionality.